meta-mender-core/recipes-mender/mender-client/mender-client.inc

```diff
@@ -98,6 +98,7 @@ PACKAGECONFIG ?= "${@mender_feature_is_enabled("mender-client-install", "modules
 PACKAGECONFIG_append = "${@bb.utils.contains('DISTRO_FEATURES', 'mender-client-install', ' mender-client-install', '', d)}"
 PACKAGECONFIG_append = "${@bb.utils.contains('DISTRO_FEATURES', 'mender-uboot', ' u-boot', '', d)}"
 PACKAGECONFIG_append = "${@bb.utils.contains('DISTRO_FEATURES', 'mender-grub', ' grub', '', d)}"
+PACKAGECONFIG_append = "${@bb.utils.contains('DISTRO_FEATURES', 'mender-network-scripts', 'mender-inventory-network-scripts', '', d)}"
```

Comment:
Instead, of doing this, locate the line 5 lines prior and do this:

```diff
-PACKAGECONFIG ?= "${@mender_feature_is_enabled("mender-client-install", "modules", "", d)}"
+PACKAGECONFIG ?= "${@mender_feature_is_enabled("mender-client-install", "modules inventory-network-scripts", "", d)}"
```

Note that I switched `mender-inventory-network-scripts` for `inventory-network-scripts`. There is no need to prefix here, since package configs are localized to the recipe, and we are already inside `mender-client`.